pkg/chunk/grpc/table_client.go

```diff
@@ -84,12 +84,13 @@ func (c *TableClient) UpdateTable(ctx context.Context, current, expected chunk.T
 }
 
 func (c *TableClient) CreateTable(ctx context.Context, desc chunk.TableDesc) error {
-	tableDesc := &TableDesc{}
-	tableDesc.Name = desc.Name
-	tableDesc.ProvisionedRead = desc.ProvisionedRead
-	tableDesc.ProvisionedWrite = desc.ProvisionedWrite
-	tableDesc.Tags = desc.Tags
-	tableDesc.UseOnDemandIOMode = desc.UseOnDemandIOMode
+	tableDesc := &CreateTableRequest{}
```

Comment:
Please rename `tableDesc` to `req`, otherwise below it's a bit weird reading `tableDesc.Desc.*`.